pkg/users/password-dialogs.js

```diff
@@ -194,7 +194,7 @@ export function set_password_dialog(account, current_user) {
 
         return password_quality(password, force)
                 .catch(ex => {
-                    errs.password = (ex.message || ex.toString()).replace("\n", " ");
+                    errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17916-20221115-164846-1bf8b990-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)